packages/docs/docs/self-hosting/rds-parameters.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebar_position: 100
+---
+
+# Update Postgres Parameters
+
+Postgres exposes a wide variety of configuration settings that can be tuned to improve database performance. These
+settings can be changed via the CDK, which creates and associates the database cluster with an AWS Parameter Group
+containing the settings. This process requires a database restart to ensure all settings are applied, necessitating
+the special procedures below to avoid any downtime.
+
+:::caution
+
+This is a complex, multi-step process, and requires high proficiency with AWS, Postgres, and command line tools.
+
+Medplum strives to make this as easy as possible, but despite our best efforts, it is still challenging.
+
+If you have any questions, please [contact us](mailto:hello@medplum.com) or [join our Discord](https://discord.gg/medplum).
+
+:::
+
+:::tip
+
+If you are new to AWS CDK, we strongly recommend reading [Getting started with the AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html).
+
+:::
+
+## Postgres Parameters
+
+The settings contained in the Parameter Group are passed directly to Postgres, so nearly[^1]
+any [Postgres configuration parameter][postgres-config] can be set this way.
+
+[postgres-config]: https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraPostgreSQL.Reference.ParameterGroups.html#AuroraPostgreSQL.Reference.Parameters.Cluster
+
+[^1]: Notably, `log_line_prefix` [cannot be changed](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_LogAccess.Concepts.PostgreSQL.overview.parameter-groups.html#USER_LogAccess.Concepts.PostgreSQL.Log_Format.log-line-prefix)
+
+### Example: `auto_explain`
+
+As an example, one might wish to configure the `auto_explain` module to automatically log slow database queries and
+their execution plans for analysis. The module must be added to the preload list, and configuration values must be added
+to control its behavior:
+
+```js
```

Comment:
We typically use `json` rather than `js` when adding comments, just to keep VS Code happy.  Not sure if that's still relevant.